bundles/core/src/test/java/com/adobe/cq/wcm/core/components/internal/models/v1/ComponentFilesImplTest.java

```diff
@@ -65,6 +65,51 @@ void testGetPathWithOneResourceType() {
         assertEquals(expected, componentFiles.getPaths());
     }
 
+    @Test
+    void testGetPathWithOneResourceTypeAndInheritance() {
```

Comment:
it would be nice to add similar tests in `ClientLibrariesImplTest`.

Reply:
Done in 2b5b20d